rahuldkjain/github-profile-readme-generator
# feature: add telegram in the social options

## What type of PR is this? (check all applicable)


## Description

- simply add the Telegram social option with the icon

## Related Tickets & Documents

issue number: #628

## QA Instructions, Screenshots, Recordings

- It's important to notice that the asset links baseUrl is from rahuldkjain's GitHub, which means that you may not be able to see the telegram icon when you test (I included the .svg file). If I'm doing something wrong, or if a dynamic baseUrl is desirable, make it a issue and I'll be sure to implement. <3

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._



## Added to documentation?
